WalletWasabi/BitcoinCore/Processes/PidFile.cs

```diff
@@ -38,24 +45,30 @@ public PidFile(string dataDir, Network network)
 			return int.Parse(pidString);
 		}
 
-		public async Task SerializeAsync(int pid)
+		/// <summary>
+		/// Writes <paramref name="pid"/> to <see cref="FilePath"/>.
+		/// </summary>
+		/// <param name="pid">Process ID.</param>
+		public async Task WriteFileAsync(int pid)
 		{
-			IoHelpers.EnsureContainingDirectoryExists(FilePath);
 			await File.WriteAllTextAsync(FilePath, pid.ToString()).ConfigureAwait(false);
 		}
 
+		/// <summary>
+		/// Deletes PID file, if it still exists.
```

Comment:
Maybe better to say `Tries to delete PID file, if it still exists.`

Reply:
```suggestion
		/// Tries to delete PID file, if it still exists.
```